pyproject.toml

```diff
@@ -40,7 +43,27 @@ dev = [
     "pytest",
     "pytest-cov",
 ]
-xlsx = ["pandas"]
+tabular = [
+    "pyyaml",
+]
+excel = [
+    "openpyxl",
+]
+cli = [
+    "typer[all]",
+]
+all = [
+    "openpyxl",
+    "pyyaml",
+    "typer[all]",
+]
+
+[project.urls]
+Home-page = "https://github.com/alliander-opensource/power-grid-model-io"
+
+[project.scripts]
+convert = "power_grid_model_io.cli.convert:app"
```

Comment:
the script command will be inject into the environment. I would call them with a prefix like `pgm_convert` or `pgm_valdiate`. Otherwise we may have a name conflict with other system command.

Reply:
Yes, fair point. I'll fix it.